OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -828,6 +844,22 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     thread_name.set_timestamp_ns(event->GetTimestamp());
     listener_->OnThreadName(std::move(thread_name));
 
+  } else if (is_tracepoint) {
+    absl::flat_hash_map<uint64_t, orbit_grpc_protos::TracepointInfo>::const_iterator it =
+        stream_id_to_tracepoint_.find(stream_id);
+
+    if (it != stream_id_to_tracepoint_.end()) {
+      auto event = ConsumeTracepointEventPidTipCpuTime(ring_buffer, header);
+
+      orbit_grpc_protos::TracepointEvent tracepoint_event;
+      tracepoint_event.set_pid(event->GetPid());
+      tracepoint_event.set_tid(event->GetTid());
+      tracepoint_event.set_time(event->GetTimestamp());
+      tracepoint_event.set_cpu(event->GetCpu());
+
+      /*TO DO: Sending of the event*/
```

Comment:
"TO DO" should be "TODO", so that IDEs highlight it.

Reply:
Thank you!